### ISSUE TITLE
Add warning for browsers who don't support localStorage offline

### ISSUE THREAD

**Issue by OscarGodson** [OscarGodson/EpicEditor#10]
Firefox won't let you save localStorage via the `file:///` protocol so give a warning. We can do something ugly for now like an alert: 

```
alert("Your browser doesn't support storing content via your file://. Please run epiceditor via http(s):// or use a different browser.")
```

**Comment by OscarGodson** [comment]
Also, add it to the docs too.

**Comment by OscarGodson** [comment]
This can be closed because of #137 - if localStorage isn't enabled we simply write to a plain JS object and data is lost between refreshes, but no errors occur.
